typescript/src/lib/database/entities/GuildEntity.ts

```diff
@@ -80,26 +80,50 @@ export class GuildEntity extends BaseEntity implements IBaseEntity {
 	@Column('varchar', { name: 'channels.farewell', nullable: true, length: 19 })
 	public channelsFarewell?: string | null;
 
-	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.Member, type: 'textchannel' })
-	@Column('varchar', { name: 'channels.logs.member', nullable: true, length: 19 })
-	public channelsLogsMember?: string | null;
-
-	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.Message, type: 'textchannel' })
-	@Column('varchar', { name: 'channels.logs.message', nullable: true, length: 19 })
-	public channelsLogsMessage?: string | null;
-
 	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.Moderation, type: 'textchannel' })
 	@Column('varchar', { name: 'channels.logs.moderation', nullable: true, length: 19 })
 	public channelsLogsModeration?: string | null;
 
-	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.NsfwMessage, type: 'textchannel' })
-	@Column('varchar', { name: 'channels.logs.nsfw-message', nullable: true, length: 19 })
-	public channelsLogsNsfwMessage?: string | null;
-
 	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.Image, type: 'textchannel' })
 	@Column('varchar', { name: 'channels.logs.image', nullable: true, length: 19 })
 	public channelsLogsImage?: string | null;
 
+	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.MemberAdd, type: 'textchannel' })
+	@Column('varchar', { name: 'channels.logs.member-add', nullable: true, length: 19 })
+	public channelsLogsMemberAdd?: string | null;
+
+	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.MemberRemove, type: 'textchannel' })
+	@Column('varchar', { name: 'channels.logs.member-remove', nullable: true, length: 19 })
+	public channelsLogsMemberRemove?: string | null;
+
+	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.MemberNickNameUpdate, type: 'textchannel' })
+	@Column('varchar', { name: 'channels.logs.member-nickname-update', nullable: true, length: 19 })
+	public channelsLogsMemberNickNameUpdate?: string | null;
+
+	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.MemberUserNameUpdate, type: 'textchannel' })
+	@Column('varchar', { name: 'channels.logs.member-username-update', nullable: true, length: 19 })
+	public channelsLogsMemberUserNameUpdate?: string | null;
+
+	@ConfigurableKey({ description: LanguageKeys.Settings.Channels.Logs.MemberRoleUpdate, type: 'textchannel' })
+	@Column('varchar', { name: 'channels.logs.member-role-update', nullable: true, length: 19 })
+	public channelsLogsMemberRoleUpdate?: string | null;
```

Comment:
```suggestion
	@Column('varchar', { name: 'channels.logs.member-roles-update', nullable: true, length: 19 })
	public channelsLogsMemberRolesUpdate?: string | null;
```

See earlier comment about this key in C# code